packages/commonwealth/client/scripts/views/components/snapshot_proposal_selector/snapshot_proposal_selector_item.tsx

```diff
@@ -16,13 +17,13 @@ const SnapshotProposalSelectorItem = ({
   return (
     <div className="proposal-item" onClick={() => onClick(snapshot)}>
       <div className="selected">{isSelected && <CWCheck />}</div>
-      <div className="text">
-        <div className="proposal-item-text" title={snapshot.title}>
```

Comment:
remove `proposal-item-text` and `proposal-item-subtext` from css files if not used anymore